hubstaff-export.rb

```diff
@@ -32,29 +33,275 @@
 require 'date'
 require 'net/http'
 require 'fileutils'
+require 'forwardable'
+require 'base64'
 require 'json'
 require 'pp'
 
-class Hash
-  def compact
-    delete_if {|k,v| v.nil? }
+# a network error occurred
+class NetworkError < StandardError; end
+
+# an error from a command
+class CommandError < StandardError; end
+
+# an error from a command
```

Comment:
This and the previous class has the same comment. Does `TokenError` serve a different purpose?